roles/ceph-update/tasks/ceph_owner.yml

```diff
@@ -20,4 +20,4 @@
   when: "'ceph_osds_ssd' in group_names"
 
 - name: correct ceph dir owner
-  file: path=/var/lib/ceph state=directory owner=ceph group=ceph recurse=yes
+  script: files/ceph_owner.sh
```

Comment:
ansible `file` module is till very slow even if we we have prep chown. It's about 3 minutes. (10499GB USED in ceph)
Our script can do it in less than 10 seconds.